### PR TITLE
Mostly Wording changes

### PR DESCRIPTION
**Pull request by smellems**
Edits by @LeighGardner.  Some changes by me.

### CODE REVIEW

File: _pages/en/strategy-open-first.md [hunk 1]
```diff
@@ -55,97 +59,100 @@ All software licensed under an [Open Source Initiative approved license](https:/
...
+The [Minister of Employment, Workforce Development and Disability Inclusion](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-employment-workforce-development-and-disability-inclusion) and the [Minister of Families, Children and Social Development](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-families-children-and-social-development-mandate-letter) mandate letters outlined the following top priorities:
 
-- Commitment to govern in a positive, open and collaborative way.
-- Continue to raise the bar on openness, effectiveness and transparency in government.
-- This means a government that is open by default.
-- It means better digital capacity and services for Canadians.
+- commitment to govern in a positive, open and collaborative way;
+- continue to raise the bar on openness, effectiveness and transparency in government;
+  - this means a government that is open by default; and
```
Comment (gcharest) [inline _pages/en/strategy-open-first.md:142]: I think the indent at lines 142 and 143 isn't voluntary?

File: _pages/en/strategy-open-first.md [hunk 2]
```diff
@@ -283,150 +293,148 @@ The [Target Solution Delivery Model](https://sara-sabr.github.io/ITStrategy/stra
...
 
 This strategy focuses on transformative and innovative aspects, and recognises open standards and OSS as catalysts for change.
-In addition, it leverages initiatives in other GC departments/agencies and helps shape the conditions for OSS development and government innovation.
+In addition, it leverages initiatives in other GC departments/agencies to help shape the conditions for open standards, OSS and government innovation.
 This section sets out the key reasons and principles underlying the new strategy.
 
 ### Progress towards digital autonomy
 
-Change is a constant ESDC has to continuously take into account when acquiring and developing software based solutions to offer services internally and to citizens.
+Change is constant at ESDC, and as a result, the acquisition and development of IT solutions must consider the flexibility needed to keep pace with legislative changes as well as emerging technologies to offer services that are alike commercial services (e.g. banks, social media, ..) internally and to citizens.
```
Comment (gcharest) [inline _pages/en/strategy-open-first.md:318]: Are the two lone periods at the end of the parenthesis a normal thing?

File: _pages/en/strategy-open-first.md [hunk 3]
```diff
@@ -440,107 +448,110 @@ The GC [Guide for using OSS](https://www.canada.ca/en/government/system/digital-
...
 Ideally, this transformation will go beyond IT, so that we apply open methods when working together and sharing information.
 
-Open standards and OSS offer practical tools and methods for teams to work together and co-create solutions, to share their expertise freely in order to obtain better results and have their solutions in actual use sooner.
-In these teams, which sometimes involve several branches within ESDC, developers look at software code written by their colleagues and try to insert new components.
-They combine parts, borrow tools and recipes, and together build and extend a collection of solutions they can use to run digital government solutions.
+Open standards and OSS offer practical tools and methods for teams to work together and co-create solutions, to share their expertise freely to obtain better results and have their solutions in actual use earlier.
+In these teams, developers look at software code written by colleagues and try to insert new components.
+They combine parts, borrow tools and recipes, and build an extended a collection of solutions to run digital government solutions.
```
Comment (gcharest) [inline _pages/en/strategy-open-first.md:460]: >  build an extended a collection

I think the "a" was to be removed

**Review by gcharest — APPROVED** [review]
Looks great! J'ai noté quelques petits points à valider mais sinon c'est super!

File: _pages/en/strategy-open-first.md (hunk 1 above)
Comment (smellems) [reply _pages/en/strategy-open-first.md:142]: I think its ok like this.

In the mandate letters it's 2 separate "priorities"

> That starts with a commitment to govern in a positive, open and collaborative way.

and

> I also expect us to continue to raise the bar on openness, effectiveness and transparency in government.
This means a government that is open by default.
It means better digital capacity and services for Canadians.

**Pull request merged**